src/main/java/com/jmb/springfactory/model/dto/WorkLogDto.java

```diff
@@ -0,0 +1,17 @@
+package com.jmb.springfactory.model.dto;
+
+import javax.validation.constraints.NotNull;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class WorkLogDto extends BaseDto {
+
+  private static final long serialVersionUID = 6369266544916427726L;
+
+  @NotNull
+  private Double hoursWorked;
+  private CommentDto comment;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "comment" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)